packages/contracts/contracts/examples/UnirepVoting.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+import {Unirep} from '.././Unirep.sol';
+
+// Uncomment this line to use console.log
+// import 'hardhat/console.sol';
+import {ReputationVerifierHelper} from '../verifierHelpers/ReputationVerifierHelper.sol';
+import {EpochKeyVerifierHelper} from '../verifierHelpers/EpochKeyVerifierHelper.sol';
+import {VotingPrizeNFT} from './VotingPrizeNFT.sol';
+
+enum Option {
+    UP,
+    DOWN
+}
+
+contract UnirepVoting {
+    Unirep public unirep;
+    ReputationVerifierHelper public helper;
+    EpochKeyVerifierHelper public epochKeyHelper;
+
+    int[] public scores;
+    uint public immutable numTeams;
+    int public winnerScore;
+    uint[][] public projectData;
+    bool foundWinner = false;
+    VotingPrizeNFT public nft;
+
+    mapping(uint256 => uint256[]) public participants;
+    mapping(uint256 => uint256) public counts;
+    mapping(uint256 => uint256) public voted;
+    mapping(uint256 => bool) claimed;
+
+    constructor(
+        Unirep _unirep,
+        ReputationVerifierHelper _helper,
+        EpochKeyVerifierHelper _epochKeyHelper,
+        VotingPrizeNFT _nft,
+        uint8 _numTeams,
+        uint48 _epochLength
+    ) {
+        // set unirep address
+        unirep = _unirep;
+
+        // set verifier address
+        helper = _helper;
+        epochKeyHelper = _epochKeyHelper;
```

Comment:
if there are two helpers, I think reputation helper should be called `_repHelper`